marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/entity-services/testCleanMapping.sjs

```diff
@@ -0,0 +1,26 @@
+declareUpdate();
+const test = require("/test/test-helper.xqy");
+
+
+let assertions = [];
+
+assertions.push(test.assertTrue(fn.head(xdmp.eval(
+    "fn.docAvailable('/mappings/OrdersMapping/OrdersMapping-1.mapping.xml') && fn.docAvailable('/mappings/OrdersMapping/OrdersMapping-1.mapping.xml.xslt')",
+    {},{database:xdmp.modulesDatabase()}
+  ))));
+
+
+xdmp.eval('xdmp.documentDelete("/mappings/OrdersMapping/OrdersMapping-1.mapping.json")',
```

Comment:
This is duplicating the code in the trigger - can you just call xdmp.eval on the trigger instead?

Reply:
So, when mapping is deleted, it calls the trigger module 'cleanUpMapping.sjs' which removes the corresponding xml/xslt. Here, I am deleting the mapping, so that the trigger ('cleanUpMapping.sjs' ) is called. I am not duplicating the trigger code here